dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/corn/CronUtils.java

```diff
@@ -16,14 +16,13 @@
  */
 
 package org.apache.dolphinscheduler.service.corn;
-
+import static org.apache.dolphinscheduler.common.Constants.CMDPARAM_COMPLEMENT_DATA_SCHEDULE_DATE_LIST;
```

Comment:
There needs to be a blank line in front of this according to checkstyle.

Reply:
ok